src/components/user/createAppointment/BreadCrumbs/Breadcrumb.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { StyledItem, StyledLi, StyledList } from './Breadcrumb.styled';
+
+const Breadcrumb = ({ crumbs, selected }) => {
+  let disabled;
+
+  function isLast(index) {
+    return index === crumbs.length - 1;
+  }
+
+  return (
+    <nav>
+      <StyledList>
+        {crumbs.map((crumb, i) => {
+          disabled = isLast(i);
```

Comment:
You dont need global variable disabled to handle disabled on your StyledItem, the third parm in map is array, you can use .length in cicle. disabled={ crumbIndex === crumbs.length - 1 }